themes/default/content/docs/reference/service-api/_index.md

```diff
@@ -0,0 +1,1465 @@
+---
+title: "Pulumi Service API"
+meta_desc: An overview of the Pulumi Service API for querying Organization, Stack, State, etc. information.
+---
+
+The Pulumi Service API is used by the Pulumi CLI and the Pulumi Console to query and interact with state information, history, stack tags, etc. This API is available for end users to integrate into their own automation use cases.
+
+{{% notes "info" %}}
+While the Pulumi Service API is used by the Pulumi CLI and Pulumi Console, Pulumi makes no guarantees about changes that might affect backwards compatibility or cause breaking changes.
+{{% /notes %}}
+
+## Authentication
+
+All requests must be authenticated using a token via the `Authorization` HTTP header. If the token is missing or invalid, the Pulumi Service API will return a 4xx status code.
+
+The `Authorization` header must be in the form below with the literal string `token`, then a space, then your access token value.
+
+```
+Authorization: token {token}
+```
+
+{{% notes "info" %}}
```

Comment:
Given that we added the `pul-` a while ago, it might be OK to just remove this. Or phrase it like "Recently created Pulumi access tokens have a `"pul-"` prefix...

Also, in case you were wondering, you _can_ omit the `pul-` prefix and it will work just fine. (But we do want to encourage people to keep that prefix, so that automated scanners can look for tokens accidentally checked into source control and inform us so that we can automatically revoke them. e.g. GitHub does this.)

Reply:
I think I saw this note elsewhere in our docs or maybe a blog post, but since that was 2 years ago (from yesterday!) I'll remove.